node-tests/test.js

```diff
@@ -257,4 +258,38 @@ describe('included()', function() {
       expect(cacheKey).to.equal(expectedCacheKey);
     });
   });
+
+  describe('different instances of inline precompile for the same parent', function() {
+    it('should have only 1 inlinePrecompile registered for parallel babel', function() {
+      let InlinePrecompile1  = defaultsDeep( {}, require('../'))
+      let InlinePrecompile2  = defaultsDeep( {}, require('../'))
+      parent = { options: { babel: { plugins: [] } } };
+      InlinePrecompile1.parent = parent;
+      InlinePrecompile2.parent = parent;
+
+      InlinePrecompile1.included();
+      configuredPlugins = parent.options.babel.plugins;
+      expect(configuredPlugins.length).to.eql(1);
+      InlinePrecompile2.included();
+      configuredPlugins = parent.options.babel.plugins;
+      expect(configuredPlugins.length).to.eql(1);
+    });
+
+    it('should have only 1 inlinePrecompile registered for non parallel babel', function() {
+      registry.add('htmlbars-ast-plugin', nonParallelPlugin);
```

Comment:
Just to clarify, this plugin is added specifically to force us _out_ of parallel mode?

Reply:
yes. Because `_isBabelPluginRegistered` handles both cases.